src/services/ui/e2e/tests/home/index.spec.ts

```diff
@@ -1,21 +1,33 @@
 import { test, expect } from "@playwright/test";
-import * as $ from "@/selectors";
+
+const password = process.env.VITE_BOOTSTRAP_USERS_PW!;
 
 test("has title", async ({ page }) => {
   await page.goto("/");
-
-  // Expect a title "to contain" a substring.
   await expect(page).toHaveTitle(/CMS MAKO/);
 });
 
-test("get issues link", async ({ page }) => {
-  const navSelectors = new $.NavSelectors(page);
+test("has faq Page", async ({ page }) => {
+  await page.goto("/");
+  await page.getByRole("link", { name: "FAQ" }).click();
```

Comment:
there should be some sort of expectation here. like once we get to the faq page is there something we can check to make sure the page loaded correctly.